solr/core/src/test/org/apache/solr/cloud/SplitShardTest.java

```diff
@@ -89,15 +93,7 @@ public void doTest() throws IOException, SolrServerException {
             .setShardName("shard1");
     splitShard.process(cluster.getSolrClient());
     waitForState(
-        "Timed out waiting for sub shards to be active. Number of active shards="
-            + cluster
-                .getSolrClient()
-                .getClusterState()
-                .getCollection(COLLECTION_NAME)
-                .getActiveSlices()
-                .size(),
-        COLLECTION_NAME,
-        activeClusterShape(6, 6));
+        "Timed out waiting for 6 shards after split", COLLECTION_NAME, activeClusterShape(6, 6));
```

Comment:
Previous code was querying for shard count as if that would be run at failure time, which isn't the case since it's just concatenated into a string before the waiting/predicate runs, so made more accurate and simple.